src/Card.js

```diff
@@ -0,0 +1,8 @@
+export function Card({data:{setup,punchline}}){
```

Comment:
Here you could have just destructured as follows:
`export function Card({ setup, punchline })`